tests/unit_test/data_processor/data_processor_test.py

```
@@ -2114,14 +2129,40 @@
+                              "&& location.endsWith('i')) || location.length() > 20) " \
+                              "{return true;} else {return false;}"
+        FormValidationAction(
+            name="form_validation_action",
+            slot="location",
+            validation_semantic=semantic_expression,
+            bot="testing_bot",
+            user="user"
+        ).save()
 
     def test_get_slot_actions(self, save_actions):
         processor = MongoProcessor()
```

Ruff

Redefinition of unused `processor` from line 14

(F811)